src/layout.js

```diff
@@ -505,6 +505,7 @@ export function applyStaticLayout(element) {
     setStyle(element, 'height', dev().assertString(height));
   } else if (layout == Layout.RESPONSIVE) {
```

Comment:
I think this also applies to intrinsic. And I think we'll need to handle transformed pages: https://github.com/ampproject/amphtml/pull/28482/files#diff-677d382da9f8d81f61d50af24f937b32R353-R354

Reply:
I'm missing the point of the link, can you elaborate on handling transformed pages?

Reply:
Transformed pages have the sizer element already embedded in the element, we don't manually generate it. The link is where we handle grabbing the pre-rendered sizer. We'd need to set the `slot` attribute in that path, too.

Reply:
Oh hm, when I click on the link it doesn't appear to lead anywhere. I see now it references L353-355 so went to look at that snippet manually. Does it make sense to set the attribute on the found element in `layout.js` or to modify the transform where it happens such that the pre-rendered sizer it provides already comes with the `slot` attribute?

Reply:
We can do both, but there's no guarantee that the transformed output will be up-to-date.